mta/widgetastic/__init__.py

```diff
@@ -253,3 +254,12 @@ def fill(self, filepath):
     def is_displayed(self):
         self.browser.set_attribute("style", "position", self)
         return self.browser.is_displayed(self)
+
+
+class AddButton(Button):
+    """Multiple buttons with same name are present in UI.
+       So need to specify the locator.
+    """
+
+    def __locator__(self):
+        return ".//button[text()='Cancel']/following-sibling::button"
```

Comment:
btw its hack

Reply:
Yea there two buttons with same name so I had to do this , But this UI is going to be replaced with new UI soon.